Documentation/api.md

```diff
@@ -983,166 +982,155 @@ AlertmanagerConfigList is a list of AlertmanagerConfig.
 
 ## AlertmanagerConfigSpec
 
-
+AlertmanagerConfigSpec is a specification of the desired behavior of the Alertmanager configuration. By definition, the Alertmanager configuration only applies to alerts that
 
 | Field | Description | Scheme | Required |
 | ----- | ----------- | ------ | -------- |
-| route |  | *[Route](#route) | false |
-| receivers |  | [][Receiver](#receiver) | false |
-| inhibitRules |  | [][InhibitRule](#inhibitrule) | false |
+| route | The Alertmanager route definition for alerts matching the resource’s namespace. It will be added to the generated Alertmanager configuration as a first-level route. | *[Route](#route) | false |
+| receivers | List of receivers. | [][Receiver](#receiver) | false |
+| inhibitRules | List of inhibition rules. The rules will only apply to alerts matching the resource’s namespace. | [][InhibitRule](#inhibitrule) | false |
 
 [Back to TOC](#table-of-contents)
 
 ## HTTPConfig
 
-
+HTTPConfig defines a client HTTP configuration.
```

Comment:
Can we add linkio to docs here as well? WDYT? https://prometheus.io/docs/alerting/latest/configuration/#http_config

Reply:
done